src/middlewared/middlewared/plugins/disk_/sync.py

```diff
@@ -249,12 +251,12 @@ def sync_all(self, job):
             for delete in deleted:
                 self.middleware.send_event('disk.query', 'CHANGED', id=delete, cleared=True)
 
-        if licensed:
+        if licensed and status == 'MASTER':
```

Comment:
Will this ever be `BACKUP` here ?

Reply:
Huh? If it's `BACKUP` we return early up above?

Reply:
Exactly, just curious about the changes as to if standby will never reach here what's the motivation behind this conditional ?